src/eval/execute.rs

```diff
@@ -178,6 +186,20 @@ fn generate_result_from_response(resp: Response, channel: Channel, is_private: b
     }
 }
 
+async fn paste_to_pb(client: &Client, code: &str) -> reqwest::Result<String> {
+    let resp = client
+        .post("https://paste.mozilla.org/api/")
+        .form(&[
+            ("lexer", "rust"),
+            ("expires", "2073600"),
```

Comment:
By default it is `2592000` (a month). I don't think you need to modify this value manually.